CHANGELOG.md

```diff
@@ -2,7 +2,8 @@
 
 ## [Unreleased]
 
-- No changes yet.
+- Fix incorrect builds for directory input when it is a subdirectory in a v2 workspace and contains
```

Comment:
I can't understand what this means as a user, can you clarify a bit more? cc @doriable

Reply:
Done